src/modules/Dashboard/features/Applet/Schedule/Legend/Legend.tsx

```diff
@@ -74,6 +76,11 @@ export const Legend = ({ legendEvents }: LegendProps) => {
     }
   };
 
+  const handleExportScheduleSubmit = async () => {
```

Comment:
could you move it to exportSchedule popup?

Reply:
I think there is no need to do this.